readthedocs/embed/views.py

```diff
@@ -165,10 +165,7 @@ def get(self, request):
 
         # Update doc from path
         if path:
-            if path.endswith('/'):
-                doc = doc.path.rstrip('/')
-            else:
-                doc = path.split('.html', 1)[0]
+            doc = re.sub(r'(.+)\.html$', r'\1', path.strip('/'))
```

Comment:
I think we should use `regex` module here since the path comes from user's input.

Reply:
Also, this line could have a comment explaining what it does plus an example.

Reply:
a regex is dangerous when the pattern comes from the user, not the string to be matched.

Reply:
ha! I always get confused with this 😅